src/C-interface/distributed2d/bml_diagonalize_distributed2d_typed.c

```diff
@@ -237,6 +237,24 @@ void TYPED_FUNC(
     char jobz = 'V';
     char uplo = 'U';
     int info;
+    //LOG_INFO("lwork=%d\n",lwork);
+    lwork *= 2;                 // increase lwork to work around ScaLapack possible bug
+    REAL_T *work = bml_allocate_memory(lwork * sizeof(REAL_T));;
+
+    //// get lwork value from ScaLapack call
+    //// for verification
+    //lwork=-1;
+    //SYEVD(&jobz, &uplo, &m, atmp, &ione, &ione, desc, ev,
+    //      pzmat, &ione, &ione, desc, work, &lwork,
+#if defined(SINGLE_COMPLEX) || defined(DOUBLE_COMPLEX)
+    //      rwork, &lrwork,
```

Comment:
Is this comment inside an `ifdef` on purpose? If the commented lines are meant as documentation I would also comment out the `ifdefs` to make that more clear.

Reply:
ifdef commented out now

Reply:
Thanks!